app/models/user.rb

```diff
@@ -3,4 +3,11 @@ class User < ActiveRecord::Base
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable, :confirmable
+         
+  has_many :stories
+  has_many :collections, dependent: :destroy
+  
+  def collection_for(story)
+    collections.where(story_id: story.id).first
```

Comment:
This is based on the Bloccit `favorite_for` method. Basically it creates an array if the user has added a story (based on `story_id`) to their collection, and shows as nil if they have not. Should I use a different method?